rules/mockery-to-prophecy/src/MockeryUtilsTrait.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\MockeryToProphecy;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\StaticCall;
+
+trait MockeryUtilsTrait
+{
+    private function isCallToMockery(Node $node)
+    {
+        return $node instanceof StaticCall && $this->isName($node->class, 'Mockery');
+    }
+}
```

Comment:
Traits are very easy to clutter any code, so better avoid them.

Here I'd create `AbstractMockeryRector`, as in `AbstractPHPUnitRector`: https://github.com/rectorphp/rector/blob/69691157bdb0a0a27e8172925dd2bd7922e3bb29/src/Rector/AbstractPHPUnitRector.php#L14

Class is easier to refactor, has single point of origin and is easier to statically analyse.